streams/jvm/src/test/scala/zio/stream/StreamSpec.scala

```diff
@@ -776,7 +776,7 @@ class ZStreamSpec(implicit ee: org.specs2.concurrent.ExecutionEnv)
       Stream(1, 2, 3)
         .spaced(Schedule.recurs(1))
         .run(Sink.collectAll[Int])
-        .map(_ must_=== List(1, 1, 2, 2, 3, 3))
+        .map(_ must_=== List(1, 1, 2, 2, 2, 2, 3, 3, 2))
```

Comment:
Could you add a test that uses a schedule with a different output type?

Reply:
Good idea - this one requires (at least I required) some time to find out why such output sequence...
Will try to come up with some more meaningful test for this.